Cargo.toml

```diff
@@ -20,3 +20,5 @@ codegen-units = 1
 
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
+zcash_note_encryption = { path = "components/zcash_note_encryption" }
+hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "576683b9f2865f1118c309017ff36e01f84420c9" }
```

Comment:
This should probably be moved to the `zcash` org. Does not block.

Reply:
My hope was that after we upgrade to secp256k1 v0.21, that I could just upstream that upgrade. I didn't want to suggest two such upgrades upstream in rapid succession.